hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -671,23 +671,21 @@ private void scanJobs() {
         if (!shouldStartJobs()) {
             return;
         }
-
         try {
             Collection<JobRecord> jobs = jobRepository.getJobRecords();
             for (JobRecord jobRecord : jobs) {
                 JobExecutionRecord jobExecutionRecord = ensureExecutionRecord(jobRecord.getJobId(),
                         jobRepository.getJobExecutionRecord(jobRecord.getJobId()));
                 if (!jobExecutionRecord.isSuspended()) {
-                    startJobIfNotStartedOrCompleted(jobRecord, jobExecutionRecord, "discovered by scanning of JobRecords");
+                    startJobIfNotStartedOrCompleted(jobRecord, jobExecutionRecord,
+                            "discovered by scanning of JobRecords");
```

Comment:
Checkstyle didn't complain about this line being too long...

Reply:
Our convention is 120 chars maximum, but CheckStyle tolerates 3 more chars.